src/components/MediaMaker/RenderControls/index.tsx

```diff
@@ -49,11 +50,14 @@ const RenderControls: React.FC<Props> = ({ inputProps, isFetching, playerRef })
     };
   }, [isCopied]);
 
-  const onCopyLinkClicked = () => {
+  const onCopyLinkClicked = async () => {
     logButtonClick('video_generation_copy_link');
     const path = getCurrentPath();
-    if (origin) {
-      clipboardCopy(path).then(() => {
+    const response = await shortenUrl(`${path}&previewMode=true`);
+
+    const url = response?.id ? `${window.location.origin}/media/${response.id}` : path;
```

Comment:
`/media` should be added to `navigation` also, we should use `getBasePath()` instead of `window.location.origin` (see `getBasePath` usage for more context).